contrib/ros/README.md

```diff
@@ -0,0 +1,138 @@
+##### Setup your keys
+    $ sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
+##### Update to Latest Software Lists
+    $ sudo apt-get update
+##### Install ROS Kinetic Desktop Full
+    $ sudo apt-get install ros-kinetic-desktop-full
+##### Initialize rosdep
+rosdep enables you to easily install system dependencies for source you want to compile and is required to run some core components in ROS.
+    
+    $ sudo rosdep init
+    $ rosdep update
+##### Install rosinstall
+    $ sudo apt-get install python-rosinstall
+#### CREATE DEVELOPMENT WORKSPACE
+##### Create ROS Workspace
+    $ mkdir -p ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src
+##### Source ROS Setup
+    $ source /opt/ros/kinetic/setup.bash
+You can also add this to you .bashrc by using
+    
+    $ echo "source /opt/ros/kinetic/setup.bash" >> ~/.bashrc
+##### Build
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ catkin_make
+#### INSTALL INTERA SDK DEPENDENCIES
+##### Install SDK Dependencies
+    $ sudo apt-get update
+    $ sudo apt-get install git-core python-argparse python-wstool python-vcstools python-rosdep ros-kinetic-control-msgs ros-kinetic-joystick-drivers ros-kinetic-xacro ros-kinetic-tf2-ros ros-kinetic-rviz ros-kinetic-cv-bridge ros-kinetic-actionlib ros-kinetic-actionlib-msgs ros-kinetic-dynamic-reconfigure ros-kinetic-trajectory-msgs ros-kinetic-rospy-message-converter
+#### INSTALL INTERA ROBOT SDK
+##### Activate the conda environment for rllab
+    $ source activate rllab3
+
+**rllab3 is your rllab conda environment's name** 
+##### Download the SDK on your Workstation
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src
+    $ wstool init .
+    $ git clone https://github.com/RethinkRobotics/sawyer_robot.git
+    $ wstool merge sawyer_robot/sawyer_robot.rosinstall
+    $ wstool update
+##### Download the dependecies 
+    $ git clone https://github.com/ros/geometry.git
+    $ cd geometry
+    $ git checkout indigo-devel
+##### Source ROS Setup
+    $ source /opt/ros/kinetic/setup.bash
+##### Build
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ catkin_make -DPYTHON_EXECUTABLE:FILEPATH=<path_to_your_rllab_env_python_interp>
+#### GONFIGURE ROBOT COMMUNICATION/ROS WORKSPACE
+##### intera.sh ROS Environment Setup
+##### Copy the intera.sh script
+The intera.sh file already exists in intera_sdk repo, copy the file into your ros workspace.
+    
+    $ cp ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src/intera_sdk/intera.sh ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+##### Customize the intera.sh script
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ gedit intera.sh
+##### Edit the 'robot_hostname' field
+##### Edit the 'your_ip' field
+##### Verify 'ros_version' field
+update: ros_version='kinetic'
+##### Save and Close intera.sh script
+##### Initialize your SDK environment
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ ./intera.sh
```

Comment:
Only executes for real robots, need to make a note of it.

Reply:
Deleted it.